src/traces/pointcloud/attributes.js

```diff
@@ -0,0 +1,124 @@
+            'Its length must be equal to or greater than the number of points.',
+            'For the best performance and memory use, create one large `indexid` typed array',
+            'that is guaranteed to be at least as long as the largest number of points during',
+            'use, and reuse it on each `Plotly.restyle()` call.'
+        ].join(' ')
+    },
+    bounds: {
+        valType: 'data_array',
+        description: [
+            'Specify `bounds` in the shape of `[xMin, yMin, xMax, yMax] to avoid looping through',
+            'the `xy` typed array.'
+        ].join(' ')
+    },
+    text: scatterglAttrs.text,
+    marker: {
+        color: {
+            valType: 'color',
+            arrayOk: false,
+            role: 'style',
+            description: [
+                'Sets the marker fill color. It accepts a specific color.',
+                'If the color is not fully opaque and there are hundreds of thousands',
+                'of points, it may cause slower zooming and panning.'
+            ].join('')
+        },
+        opacity: {
+            valType: 'number',
+            min: 0,
+            max: 1,
+            dflt: 1,
+            arrayOk: false,
+            role: 'style',
+            description: [
+                'Sets the marker opacity. The default value is `1` (fully opaque).',
+                'If the markers are not fully opaque and there are hundreds of thousands',
+                'of points, it may cause slower zooming and panning.',
+                'Opacity fades the color even if `blend` is left on `false` even if there',
+                'is no translucency effect in that case.'
+            ].join(' ')
+        },
+        blend: {
+            valType: 'boolean',
+            dflt: false,
+            role: 'style',
+            description: [
+                'Determines if colors are blended together for a translucency effect',
+                'in case `opacity` is specified as a value less then `1`.',
+                'Setting `blend` to `true` reduces zoom/pan',
+                'speed if used with large numbers of points.'
+            ].join(' ')
+        },
+        sizemin: {
+            valType: 'number',
+            min: 0.1,
+            max: 2,
+            dflt: 0.5,
+            role: 'style',
+            description: [
+                'Sets the minimum size (in px) of the rendered marker points, effective when',
+                'the `pointcloud` shows a million or more points.'
+            ].join(' ')
+        },
+        sizemax: {
+            valType: 'number',
+            min: 0.1,
+            dflt: 20,
+            role: 'style',
+            description: [
+                'Sets the maximum size (in px) of the rendered marker points.',
+                'Effective when the `pointcloud` shows only few points.'
+            ].join(' ')
+        },
+        border: {
```

Comment:
Yes, `marker.border` is different enough from `marker.line` to deserve its own name :+1: